demo-project/docs/source/conf.py

```diff
@@ -92,7 +92,7 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "sphinx_rtd_theme"
+html_theme = "kedro-sphinx-theme"
```

Comment:
we need to change this here  kedro-viz/docs/source/conf.py

Reply:
🦅 good catch!

Reply:
ah i see thanks @rashidakanchwala